package.json

```diff
@@ -24,19 +24,26 @@
     "url": "https://github.com/uphold/debugnyan.git"
   },
   "scripts": {
-    "changelog": "github_changelog_generator --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
-    "lint": "eslint index.js test",
-    "test": "jest --coverage --env=node",
-    "version": "npm run changelog --future-release=$npm_package_version && git add -A CHANGELOG.md"
+    "changelog": "uphold-scripts changelog v$npm_package_version",
```

Comment:
Needs to regenerate the changelog file to get the new format (don't forget to delete the section with the unreleased PRs).

Reply:
Nice, will regenerate.